elide-integration-tests/src/test/java/com/yahoo/elide/async/integration/tests/framework/AsyncIntegrationTestApplicationResourceConfig.java

```diff
@@ -90,10 +95,11 @@ protected void configure() {
                         .build());
                 bind(elide).to(Elide.class).named("elide");
 
-                AsyncQueryDAO asyncQueryDao = new DefaultAsyncQueryDAO(elide, elide.getDataStore());
+                AsyncQueryDAO asyncQueryDao = new DefaultAsyncQueryDAO(elide.getElideSettings(), elide.getDataStore());
                 bind(asyncQueryDao).to(AsyncQueryDAO.class);
 
-                ResultStorageEngine resultStorageEngine = new DefaultResultStorageEngine(elide, elide.getDataStore(), "http://localhost:8080");
+                ResultStorageEngine resultStorageEngine = new DefaultResultStorageEngine(elide.getElideSettings(),
+                        elide.getDataStore(), "http://localhost:8080/AsyncQueryResultStorage/");
```

Comment:
Hardcoding the port like this might make it not match the RestAssured configuration.